8.observability/README.md

```diff
@@ -0,0 +1,126 @@
+
+In this sample you will:
+
+- Deploy Zipkin and configure it as a tracing provider for Dapr.
+- Instrument an application and deploy it
+- Troubleshoot a performance issue in a distributed application.
+
+## Prerequisites
+
+This sample builds on top of the [distributed calculator](../3.distributed-calculator/README.md) sample and so requires a Dapr installation on a Kubernetes cluster as well as a state store. If you have not done so for previous samples:
+
+1. Clone this repo using `git clone https://github.com/dapr/samples.git`
+
+2. [Install Dapr on Kubernetes](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#installing-dapr-on-a-kubernetes-cluster).
+3. [Configure Redis](https://github.com/dapr/docs/tree/master/howto/configure-redis) as a state store for Dapr.
+
+## Configure Dapr tracing in the cluster
+
+Review *./deploy/tracing.yaml*:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: tracing
+spec:
+  tracing:
+    enabled: true
+    expandParams: true
+    includeBody: true
+```
+
+This configuration definition will enable Dapr to do tracing. Deploy the configuration by running:
+
+`kubectl apply -f ./deploy/tracing.yaml`
+
+## Deploy Zipkin to the cluster and set it as the tracing provider
+
+In this sample Zipkin is used as the underlying tracing system. Examine [*./deploy/zipkin.yaml*](./deploy/zipkin.yaml) and see how it includes three sections:
+
+1. A **Deployment** for Zipkin using the *openzipkin/zipkin* docker image.
+2. A **Service** which will expose Zipkin internally as a ClusterIP in Kubernetes.
+3. A **Component** that defines Zipkin as the tracing provider for Dapr.
+
+Deploy to your cluster by running:
+
+`kubectl apply -f ./deploy/zipkin.yaml`
+
+Now that Zipkin is deployed, you can access the Zipkin UI by creating a tunnel to the internal Zipkin service you just created by running:
+
+`kubectl port-forward svc/zipkin 9411:9411`
+
+On your browser go to [http://localhost:9411](http://localhost:9411). You should be able to see the Zipkin dashboard.
+
+## Instrument the application for tracing and deploy it
+
+To instrument a service for tracing with Dapr, no code changes are required - Dapr handles all of the tracing using the Dapr side-car. All that is needed is to add the Dapr annotation for tracing in the deployment yaml along with the other Dapr annotations. The annotation looks like this:
+
+```yaml
+...
+annotations:
+...
+    dapr.io/config: "tracing"
+...
+ ```
+
+For this sample, no action is needed since tracing is already enabled for every service in the distributed calculator app. You can find the annotation in each one of the calculator yaml files - For example review the yaml file for the calculator front end service [here](https://github.com/dapr/samples/blob/master/3.distributed-calculator/deploy/react-calculator.yaml#L36).
```

Comment:
Since these terms are overloaded I think this sentence

"For this sample, no action is needed since tracing is already enabled for every service in the distributed calculator app"

might be a bit clearer as:

"For this sample, no action is needed since tracing is already enabled for every app in the distributed calculator."

Reply:
Each one of the pods is a service, together they make up the calculator app. Would "microservice" help clarify this?

Reply:
Part of the problem is overloaded terms =(  We've mostly been using "app" for the process next to daprd, but I understand it can mean a higher level "collection of logic working to provide customer logic" concept as well.

And service may sound like k8s service.

I don't know if we have a solid convention, but I think for new users, being clear on what terms mean in our context helps a lot.  Would like to hear what others think for terminology.

Reply:
These are excellent points. We should try to be consistent across the various docs and samples. I realize now the "service" specifically can me confusing in the context of k8s. I need to go over other READMEs we have across the various areas and come up with the kind of terms we want to use